android/app/build.gradle

```diff
@@ -154,45 +202,45 @@ dependencies {
     * ---------------------------- */
 
     // Library as defined in settings.gradle
-    implementation project(path: ':react-native-firebase')
+    //implementation project(path: ':react-native-firebase')
 
     // Required dependencies
     //noinspection GradleCompatible
-    implementation "com.google.firebase:firebase-core:16.0.9"
-    implementation "com.google.android.gms:play-services-base:16.1.0"
+    implementation "com.google.firebase:firebase-core:17.2.0"
```

Comment:
strongly recommend going to the 'bom' style for these (and any projects you are using firebase on) like so (this is mine, I may not have all the modules desired, but the style is illustrative:

```groovy
    implementation platform("com.google.firebase:firebase-bom:22.1.0")
    implementation "com.google.firebase:firebase-core"
    implementation "com.google.firebase:firebase-auth"
    implementation "com.google.firebase:firebase-analytics"
    implementation "com.google.firebase:firebase-perf"
    implementation "com.google.firebase:firebase-config"
    implementation "com.google.firebase:firebase-functions"
    implementation "com.google.firebase:firebase-storage"
    implementation "com.google.firebase:firebase-dynamic-links"
    implementation "com.google.firebase:firebase-firestore"
    implementation "com.google.firebase:firebase-messaging"
    implementation 'com.crashlytics.sdk.android:crashlytics'


```

Reply:
I think this should be right, might want to double check. Added

implementation platform("com.google.firebase:firebase-bom:22.2.0")

And removed all versions from other dependencies.